src/dlgTriggerEditor.cpp

```diff
@@ -895,8 +895,21 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 
 void dlgTriggerEditor::slot_editorThemeChanged()
 {
+    auto pEdbee = edbee::Edbee::instance();
+    edbee::TextThemeManager* pThemeManager = nullptr;
+    edbee::TextTheme* pTheme = nullptr;
+    if (pEdbee) {
+        pThemeManager = pEdbee->themeManager();
+        if (pThemeManager) {
+            pTheme = pThemeManager->theme(mpHost->mEditorTheme);
+        }
+    }
```

Comment:
Could you add the 'why' comments here explaining why this does what it does?

Reply:
I suppose so - if the debug output following a failure to get the theme isn't clear enough...

However now it's been confirmed for other people I'm gonna put in some checks to see when/how the values are (not) set to "proper" values...